cfg/dist.js

```diff
@@ -1,42 +0,0 @@
-'use strict';
```

Comment:
This divide is not implemented yet, I have only set generate bundle.js in `dist` folder and copy there rest of files. My first goal was to create a working example, now I'm reading how to develop this.